MapboxNavigation/NetworkConfiguartion.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+class NetworkConfiguration : NSObject{
+    @objc static let sharedConfiguration = NetworkConfiguration()
+    
+    // The current API base URL of map
+    private var apiBaseURL : String?
+    
+    // The PRC base URL for Mapbox APIs other than the telemetry API.
+    private let mapboxChinaBaseAPIURL = "https://api.mapbox.cn"
+    
+    // The base URL host for Mapbox China
+    public let mapboxChinaBaseURLHost = "api.mapbox.cn"
```

Comment:
If it can make public, that is really helpful for only change the map endpoint. But for a further consideration, I wanna to make this module easily change the map endpoint, api-directions endpoint and integrate Chinese TTS service provided by Baidu.